### DISCUSSION TITLE
Feature Request: Render Markdown formatting in all views (widget, list view, etc.)

### DISCUSSION THREAD

**Discussion by nagmat84** [TechbeeAT/jtxBoard#695] [Ideas]
jxBoard supports Makdown formatting for the description attribute of tasks, notes, etc. However, the formatting is only rendered inside the app on the overview view.

If you place the jxWidget on the Android screen and enable the option to also show the description, the widget displays the raw text.

It would be nice, if all views would use Markdown rendering (of course, except for the edit view). Maybe this could be made optional, i.e. via a setting.

**Comment by patrickunterwegs** [comment]
Hey @nagmat84 , thank you for your feature request. 
For some time now I was against such an option in the list as it could break the layout eg. when you have a heading in the text. But it seems to be a popular request. I'll check it and add a setting in the list (similar to flat view in the menu).

About the widget I'm not sure if it will allow the formatting, but I'll try.
